pkgs/games/factorio/default.nix

```diff
@@ -17,7 +17,7 @@ let
   variants = {
     x64 = {
       url = "https://www.factorio.com/get-download/${version}/alpha/linux64";
-      sha256 = "1xpzrx3q678519qgjl92fxn3qv55hd188x9jp6dcfk2ljhi1gmqk";
+      sha256 = "1dc53swnlbz23czw8v3ldl4ayp7118qyja4vh1z5dsd871swz9qg";
     };
 
     x32 = {
```

Comment:
You can download the files from https://www.factorio.com/download/stable, then use the commands joachifm suggested. "nix-prefetch-url --type sha256 file://<FILENAME>" should work, and will keep you from needing to download it twice.

Oh, and also .28 is already out.